site/app/models/gradeable/Gradeable.php

```diff
@@ -92,8 +92,8 @@ class Gradeable extends AbstractModel {
     private $any_manual_grades = null;
     /** @property @var bool If any submissions exist */
     private $any_submissions = null;
-    /** @property @var bool If any teams have been formed */
-    private $any_teams = null;
+    /** @property @var Team[] Any teams that have been formed */
+    private $teams = null;
```

Comment:
So _that's_ how you hide a variable from `AbstractModel::__call`

Reply:
if you don't have the `@property`  tag on it i've had it not work, but I like to be extra sure